client/src/app/site/agenda/services/agenda-repository.service.ts

```diff
@@ -74,7 +164,9 @@ export class AgendaRepositoryService extends BaseRepository<ViewItem, Item> {
 
     public createViewModel(item: Item): ViewItem {
         const contentObject = this.getContentObject(item);
-
+        if (!contentObject) {
```

Comment:
Please remove this

Reply:
Removed now. Actually the warnings were supposed to stay until race conditions have been resolved to track them more easily.